_build/redirection_map

```diff
@@ -508,3 +508,4 @@
 /components/mailer /mailer
 /messenger/message-recorder messenger/dispatch_after_current_bus
 /components/stopwatch https://github.com/symfony/stopwatch
+/service_container/3.3-di-changes https://symfony.com/doc/3.4/service_container/3.3-di-changes.html
```

Comment:
@javiereguiluz can you maybe confirm whether this works?

(I believe the allowed full URLs are github.com and symfony.com, but I'm not sure)

Reply:
You are right. I've just double checked and we allow redirects to URL that start with either `https://symfony.com/` or `https://github.com/`